Update Travis CI badge URL

fvoges opened the pull request:
update Travis CI badge URL
To test, check that the badge is not grey

fvoges committed:
Update Travis CI badge URL

ghost commented:
Your pull request doesn't follow our guidelines. Please fix the following:

* Pull request description must include verification steps [(?)](https://gitmagic.io/rules/#/pull-request/body-must-include-verification-steps)
* Pull request description cannot be empty [(?)](https://gitmagic.io/rules/#/pull-request/body-cannot-be-empty)

[Click here](https://app.gitmagic.io/fvoges/puppet-motd/7/bbd819bd-d04c-473b-8c46-6639b496bdb7/1f2ecdbe-586c-47b4-8de1-f6916f7e3077) for details.

Thank you! :pray:
<sub>
This comment was made by [GitMagic](https://gitmagic.io/) – Magically enforcing your contribution guidelines.
</sub>

ghost commented:
This pull request still violates some of our guidelines:

* Pull request description must include verification steps [(?)](https://gitmagic.io/rules/#/pull-request/body-must-include-verification-steps)

[Click here](https://app.gitmagic.io/fvoges/puppet-motd/7/bbd819bd-d04c-473b-8c46-6639b496bdb7/1f2ecdbe-586c-47b4-8de1-f6916f7e3077) for details.

ghost commented:
Thank you, the title and description now looks good! :bowtie:

ghost commented:
This pull request still violates some of our guidelines:

* Pull request title must start with uppercase [(?)](https://gitmagic.io/rules/#/pull-request/subject-must-start-with-case)

[Click here](https://app.gitmagic.io/fvoges/puppet-motd/7/bbd819bd-d04c-473b-8c46-6639b496bdb7/1f2ecdbe-586c-47b4-8de1-f6916f7e3077) for details.

ghost commented:
Thank you, the title and description now looks good! :bowtie:

fvoges committed:
Update issue tracker URL

To test, README link for issues should point to the module issue tracker
on GitHub

fvoges committed:
Merge branch 'master' into fix_travis_ci_badge
